layouts/shortcodes/home-section-resources.html

```diff
@@ -0,0 +1,75 @@
+<!-- 
+  Copyright (c) 2019 Eclipse Foundation, Inc.
+
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+
+  Contributors:
+    Christopher Guindon <chris.guindon@eclipse-foundation.org>
```

Comment:
When creating a new file, you should put your name as the contributor not mine :)